topiary-core/src/atom_collection.rs

```diff
@@ -591,6 +609,22 @@ impl AtomCollection {
         self.append.entry(target_node.id()).or_default().push(atom);
     }
 
+    /// Indicates whether we are in a multiline context or not.
+    /// # Arguments
+    ///
+    /// * `node` - The node to which the atom applies.
+    /// # Returns
+    ///
+    /// A boolean indicating whether a given node has a a parent  labelled as multi-line.
+    /// If the provided node has no parent, the function returns `false`.
+    fn in_multiline_context(&self, node: &Node) -> bool {
```

Comment:
there seems to be a lot of variance between referring to multilines, would help to normalize for consistency:
* `multiline`
* `multi-line`
* `multi_line`